docs/UserGuide.md

```diff
@@ -78,15 +78,35 @@ Format: `help`
 
 Adds a person to the address book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `add n/NAME [{SPECIFIER}/{DATA}]`
+
+Here are all the specifiers that can be used:
+| Specifier   | Name of Field| Optional? |
+| ----------- | -----------  |
```

Comment:
This does not render the table properly, there should be a newline before the table and the number of columns should be consistent:

```suggestion
Here are all the specifiers that can be used:

| Specifier | Name of Field | Optional? |
| --------- | ------------- | --------- |
```

Also FYI, there's a `Display the rich diff` button in GitHub that can be used to see if it's formatting properly.

<img width="1084" alt="Screenshot 2023-03-05 at 10 09 53" src="https://user-images.githubusercontent.com/20805849/222937860-4ac4cc2b-997b-470e-9963-abf3cb1b9ff2.png">